worlds/trackmania/client.py

```diff
@@ -0,0 +1,258 @@
+import asyncio
+import Utils
+import websockets
+import functools
+from copy import deepcopy
+from typing import List, Any, Iterable
```

Comment:
```suggestion
from typing import Any, Iterable
```
The lowercase version of list can be used for typing now.